prisma/schema.prisma

```diff
@@ -135,13 +135,15 @@ model Exercise {
   objectives String[]
   users      ExerciseUser[]
   logs       DeliveryLog[]
+  createdAt  DateTime       @default(now())
+  updatedAt  DateTime       @default(now())
```

Comment:
`updatedAt DateTime? @updatedAt`

No need to add the @default if you do this

Reply:
updated